pkg/api/folder.go

```diff
@@ -210,8 +222,21 @@ func (hs *HTTPServer) MoveFolder(c *contextmodel.ReqContext) response.Response {
 		var theFolder *folder.Folder
 		var err error
 		if cmd.NewParentUID != "" {
+			if !hs.AccessControl.IsDisabled() {
```

Comment:
question: should we check that the user has `folders:create` permissions if they're trying to move the folder to the root?

Reply:
that would make sense as it would be a workaround to bypass not having `folders:create`, have you seen any downsides to it?

Reply:
Yeah I think that make sense